mono/mini/interp/transform.h

```diff
@@ -167,6 +167,7 @@ typedef struct
 	InterpBasicBlock *entry_bb, *cbb;
 	int bb_count;
 	MonoMemPool     *mempool;
+	MonoMemoryManager *mem_manager;
```

Comment:
Is it safe to add this reference? Are transforms tied to the lifetime of an assembly or is there any chance of one still existing after an ALC is unloaded? If you're sure it's safe I'm a big fan, this makes things much cleaner.

Reply:
TransformData is just a transient data structure used by transform.c, its the same as MonoCompile for the JIT.